### PR TITLE
Feature/brpa 883 tiktok marketing api client

### PR DESCRIPTION
**Pull request by MIzidio**
#### What is the purpose of this pull request?

Create a client for TikTok Marketing API

#### Types of changes

- [ ] Chore (non-breaking change which doesn't change any functionalities)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Comment by ConradoClark** [comment]
Build seems to be failing:
```
error TS2306: File '/cache/build/build_vtex_master/vtex.tiktok-tbp@0.3.0/05544bc3043d4a60/src/node/utils/constants.ts' is not a module.

5 import { workspace, account, appName, appId } from '../utils/constants'
```

I think there may be content missing from the `constants` file 🤔

**Review by felipefadul — APPROVED** [review]
LGTM!

File: node/clients/tikTokMarketing.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+import type { TikTokMarketingResponse } from '../typings/tikTokMarketing'
+import { workspace, account, appName, appId } from '../utils/constants'
+
+const routes = {
+  getTikTokMarketing: () =>
+    `https://business-api.tiktok.com/open_api/v1.2/oauth2/access_token/`,
+  getSecret: () =>
+    `http://app.io.vtex.com/${appName}/v0/${account}/${workspace}/_v/api/tiktok-tbp-config/secret`,
+}
```
Comment (ConradoClark) [inline node/clients/tikTokMarketing.ts:12]: I think this method should be in another client 🤔

Comment (MIzidio) [reply node/clients/tikTokMarketing.ts]: I agree, I changed the secret to be a parameter now and removed the getSecret method

**Review by ConradoClark — APPROVED** [review]

**Pull request merged**